Sources/Turf/JSON.swift

```diff
@@ -209,3 +217,39 @@ extension JSONValue: Codable {
         }
     }
 }
+
+/**
+ A type that can be represented as a `JSONValue` instance.
+ */
+public protocol JSONValueConvertible {
+    /// The instance wrapped in a `JSONValue` instance.
+    var jsonValue: JSONValue { get }
+}
+
+extension String: JSONValueConvertible {
+    public var jsonValue: JSONValue { return .string(self) }
+}
+
+extension Int: JSONValueConvertible {
+    public var jsonValue: JSONValue { return .number(Double(self)) }
+}
+
+extension Double: JSONValueConvertible {
+    public var jsonValue: JSONValue { return .number(Double(self)) }
+}
+
+extension Bool: JSONValueConvertible {
+    public var jsonValue: JSONValue { return .boolean(self) }
+}
+
+extension Array: JSONValueConvertible where Element == JSONValueConvertible? {
+    public var jsonValue: JSONValue {
+        return .array(map { $0?.jsonValue })
+    }
+}
+
+extension Dictionary: JSONValueConvertible where Key == String, Value == JSONValueConvertible? {
```

Comment:
If we did https://github.com/mapbox/turf-swift/pull/159/files#r721695103, maybe these generic constraints could be `== JSONValueConvertible`